### ISSUE TITLE
cc-oci-runtime + docker 1.13.1 - docker kill does not work

### ISSUE THREAD

**Issue by chavafg** [01org/cc-oci-runtime#692]
When trying to kill a clear container with `docker 1.13.1`, docker fails to kill the container and returns an exit code of `0`. 

```
[fuentess@cc-centos docker]$ sudo docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES
a8af0bf32c7f        ubuntu              "bash -c 'sleep 5 ..."   About an hour ago   Up About an hour                        y4Jg6eSoTqVlTx5w2f2S8gJo4vvYlvSZ
[fuentess@cc-centos docker]$ sudo docker kill a8af0bf32c7f
a8af0bf32c7f
[fuentess@cc-centos docker]$ echo $?
0
[fuentess@cc-centos docker]$ sudo docker ps
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES
a8af0bf32c7f        ubuntu              "bash -c 'sleep 5 ..."   About an hour ago   Up About an hour                        y4Jg6eSoTqVlTx5w2f2S8gJo4vvYlvSZ
```

Removing the container while is running, also sends an error message and this time the container is deleted, but `/var/lib/docker/containers` is dirty:

```
[fuentess@cc-centos docker]$ sudo docker ps                                                                                                                                                                                                            
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS               NAMES
a8af0bf32c7f        ubuntu              "bash -c 'sleep 5 ..."   About an hour ago   Up About an hour                        y4Jg6eSoTqVlTx5w2f2S8gJo4vvYlvSZ
[fuentess@cc-centos docker]$ sudo docker rm -f a8af0bf32c7f
Error response from daemon: Unable to remove filesystem for a8af0bf32c7fab46e1920c426895893d5d9ad1db9600e30dc4f5213d282b5205: remove /var/lib/docker/containers/a8af0bf32c7fab46e1920c426895893d5d9ad1db9600e30dc4f5213d282b5205/shm: device or resource busy
[fuentess@cc-centos docker]$ sudo docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
[fuentess@cc-centos docker]$ sudo ls /var/lib/docker/containers
16bfc77b7c6cef2e6edb54f1255e36331080ae350af94709b828f76cbdf71720  675f0064330fed537c1ce7602661a45a35e62202647af20c92ca6acc0c9ce742  b95064c011226146aedb1b6acf0e09160b987acff6cabd05840486e1c8f0dd14
1e288b7dcfe0fb3ce69d706d8f7f205206b23385c2dfea206acaa2b5c347e718  6a02c4fc362aab18926482071ee4da39e186d7b07b66e022ec8e60952e91b017  c0594ccbaea2bd2732df6a646fe66d4841a73c4c2279117e4d17fe31e3350b11
2bc859239d469355e3a026f58fcc6261c51da8172bbf07596adba663e5945c1d  78f2335763976b70e1be3fc59ed41e089d008fec2d719c4f5f222c8bd362c217  c0911faba38bee1f053e93fe6538f0046d28a9e04c5896658a40f9181b57ea3d
38dbf769c94d73187d714ec92ca8f40480125f07ffd81403908e71d31b12a10c  87e82833e8f6ef762846d0459161a7761649e5556c99d4699fcfd5e6992d0f28  cd6caa15c6f8ee05415ae924b2be423aef878b051199670acc4f8d185172e24f
598cda9c77b38510afc0917dfa44ae29f05dbfca4a51c469526b8ce6fe81e73a  90ad2c66d915a503c8c2aba52f30501d1a1ee47e2257bd3b52a8ae1fea5fd278  d4a0814c11760d2e233626148407c131e913b2f6b7dc2d79209d7e8e3a344479
5c2e2e89ac7b723ef6a7df998807615200e42c967ecc586cea70d66e629c302f  9a2fde5583fdab701d726aa7ed2b9732df5eea7587a79e6e51dc4eb390a4b679  dbc18bc481cf321b3a81e86d863b08067831f08647d80a720a320e1925b13805
5f955884c125f431fff209856200b3770acbcf191161677b7d1f4a60d3125634  a52c2cedfbcce866d6e56ccdf2234946bc1808724a0e0a62544ee718011e2c56  f4a6c473d3654a4f74769c1db1c060062627e996cbf8fe104c8b1bf18c524079
601143e9d3180077b1c531cb572837180131a1f382f63838f1c96cb015eb0146  a8af0bf32c7fab46e1920c426895893d5d9ad1db9600e30dc4f5213d282b5205  f92544dccc59d16694e993e60c0d966b5a55ba7ce65435461ec0ce68144e786c
60baf4226892d910243ec55ea08bff036e28b84f4cc4f5602177f0fabfab3534  ae010de16f30747db115f8f643564c62e8812c334316116b3835dfb923eae427
[fuentess@cc-centos docker]$ 



```

**Comment by jodh-intel** [comment]
See #693.

**Comment by devimc** [comment]
fixed in #776
